app.js

```diff
@@ -38,4 +38,13 @@ app.post('/notes', (req, res) => {
   res.status(200).json(newNote);
 });
 
+app.put('/notes/:id', (req, res) => {
+  const {id} = req.params
+  const note = req.body
+  const noteIndex = app.locals.notes.findIndex(note => note.id == id)
```

Comment:
Awesome 👍 We could look into using a filter() method later on if it's more direct.